src/MerchantCenter/ShoppingContentDateTrait.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Vendor\Google\Service\ShoppingContent\Date as ShoppingContentDate;
+use Datetime;
```

Comment:
Should be the right case:
```suggestion
use DateTime;
```